travis/before-install-maven

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -eux
 
-./gradlew prepareMavenBuild install -x javadoc
+./gradlew prepareMavenBuild publishToMavenLocal -x javadoc
```

Comment:
Still disable javadoc?

Reply:
Yes, that still makes sense here. This Gradle command is intended to populate `~/.m2` for use by a subsequent Maven build. The Maven build doesn’t care about documentation, so we may as well skip it.

Excluding the `javadoc` tasks here does the same thing it did before: `javadoc` jars will be installed under `~/.m2`, but these jars will be empty except for the manifest files. The important thing is that no time will be spent creating Javadoc HTML pages.